Makefile

```diff
@@ -5,7 +5,7 @@ PHONY:
 ready: PHONY Dockerfile
 	docker build --target dev --tag mare-dev .
 	docker rm -f mare-dev || echo "the mare-dev container wasn't running"
-	docker run --name mare-dev -v $(shell pwd):/opt/code -d --rm --memory 4g mare-dev tail -f /dev/null
+	docker run --name mare-dev -p 8080:8080 -v $(shell pwd):/opt/code -d --rm --memory 4g mare-dev tail -f /dev/null
```

Comment:
This made it easier for me to run the HTTP example, but I don't know if you'd prefer to keep the container isolated.

Reply:
Does it run for you if you use the `make example` target? That should run it outside the container, I think.

Reply:
I'll give it a shot!